src/backend/distributed/deparser/deparse_role_stmts.c

```diff
@@ -359,7 +359,15 @@ AppendGrantRoleStmt(StringInfo buf, GrantRoleStmt *stmt)
 			if (strcmp(opt->defname, "admin") == 0)
 			{
 				appendStringInfo(buf, "ADMIN OPTION FOR ");
-				break;
+			}
+			else if (strcmp(opt->defname, "inherit") == 0);
```

Comment:
Note that the build is failing because of the extra semicolon here
```suggestion
			else if (strcmp(opt->defname, "inherit") == 0)
```